api/lib/use-cases/FetchRecords.js

```diff
@@ -0,0 +1,32 @@
+const mergeResponses = require('../../lib/MergeResponses');
+const cleanRecord = require('../../lib/use-cases/CleanRecord')({
```

Comment:
cleanRecord is already defined in libDependencies and should be injected (added to "options")